practices/managing-node-projects.md

```diff
@@ -10,7 +12,30 @@
 * [Dependency management tools](#dependency-management-tools)
 * [Handling dependencies from third parties](#handling-dependencies-from-third-parties)
 
-The following guide describes how we manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+The following guide describes how we use node and manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+
+
+## Specifying versions of node
+
+It is important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
```

Comment:
[Use contractions](https://github.com/springernature/frontend-playbook/blob/master/writing/house-style.md#plain-english): "it's important" vs "it is important", "you're encouraged" vs "you are encouraged"